sql_generators/active_users_aggregates_v3/templates/focus_android_view.sql

```diff
@@ -5,7 +5,7 @@ CREATE OR REPLACE VIEW
 AS
 SELECT
   * EXCEPT (app_version, app_name),
-  IF(app_name IN ('Focus Android Glean', 'Focus Android'), 'Focus Android', app_name) AS app_name,
+  app_name,
```

Comment:
Instead of 

```sql
EXCEPT (app_version, app_name), 
app_name,
```

This can probably be:

```sql
EXCEPT (app_version),
```

Reply:
The order of the columns is for compatibility when building the mobile joined view `telemetry.active_users_aggregates_mobile`.